platform/admin/core/admin-core-alertservice/src/test/groovy/org/codice/ddf/admin/core/alert/service/internal/AlertServiceBeanTest.groovy

```diff
@@ -0,0 +1,264 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.core.alert.service.internal
+
+import org.codice.ddf.admin.core.alert.service.data.internal.Alert
+import org.osgi.service.event.Event
+import spock.lang.Specification
+
+import javax.management.MalformedObjectNameException
+import javax.management.ObjectName
+import java.lang.management.ManagementFactory
+
+class AlertServiceBeanTest extends Specification {
+
+    final mbeanName = new ObjectName(AlertServiceBean.class.getName() + ":service=alert-service")
+
+    def alertServiceBean
+
+    void setup() throws Exception {
+        // the keystore path must be set for the InsecureDefaultsService to use
+        System.setProperty("javax.net.ssl.keyStore", "TestKeystorePath")
+        alertServiceBean = new AlertServiceBean(ManagementFactory.getPlatformMBeanServer())
+        assert !isRegistered()
+        assert alertServiceBean.addedAlerts.isEmpty()
+    }
+
+    void cleanup() throws Exception {
+        if (isRegistered()) {
+            ManagementFactory.getPlatformMBeanServer()
+                    .unregisterMBean(mbeanName)
+        }
+    }
+
+    def 'test init'() throws Exception {
+        when:
+            alertServiceBean.init()
+
+        then:
+            isRegistered()
+    }
+
+    def 'test init when the bean has already been registered'() throws Exception {
+        when:
+            alertServiceBean.init()
+
+        then:
```

Comment:
❗️ This is already tested in the previous test so we probably could remove this `then` and combine the two `when` blocks, i.e., call `init()` twice and and make sure nothing breaks and the MBean is still registered.